5g-core-topology/Kptfile

```diff
@@ -0,0 +1,10 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: 5g-core-topology
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: >
```

Comment:
I think it would be helpful to add a note on what this kptfile could potentially contain.

Reply:
I added some text there and also a link to John's description in the "kpt" repo

Reply:
thanks